jbpm-flow/src/main/java/org/jbpm/workflow/instance/node/DynamicNodeInstance.java

```diff
@@ -74,11 +73,7 @@ public void internalTrigger(NodeInstance from, String type) {
         	agenda.getRuleFlowGroup(ruleFlowGroup).setAutoDeactivate(false);
         	agenda.activateRuleFlowGroup(ruleFlowGroup, getProcessInstance().getId(), getUniqueId());
     	}
-//    	if (getDynamicNode().isAutoComplete() && getNodeInstances(false).isEmpty()) {
-//    		triggerCompleted(NodeImpl.CONNECTION_DEFAULT_TYPE);
-//    	}
```

Comment:
why this is commented out?

Reply:
no idea.... Just removed it when I saw commented code.

Reply:
interesting I noticed it was commented out but not that it was (rightfully removed), thanks!